src/talk/renderer/TitleBar/components/MainMenu.vue

```diff
@@ -73,11 +108,19 @@ const openInWeb = () => window.open(generateUrl(getCurrentTalkRoutePath()), '_bl
 			</template>
 			{{ t('talk_desktop', 'App settings') }}
 		</NcActionButton>
-		<NcActionButton @click="showHelp">
+		<NcActionButton close-after-click @click="showHelp">
 			<template #icon>
 				<IconInformationOutline :size="20" />
 			</template>
 			{{ t('talk_desktop', 'About') }}
 		</NcActionButton>
 	</NcActions>
 </template>
+
+<style lang="scss" scoped>
```

Comment:
Although we use SCSS in many apps, including Talk, Talk Desktop doesn't use it.

```suggestion
<style scoped>
```